sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStore.java

```diff
@@ -78,6 +78,19 @@
 
 
     /**
+     * Stores the TaskStatus of a particular Task. The {@link TaskInfo} for this exact task MUST have already been
```

Comment:
if it must already have been written the name should be `updateStatus` probably.